Index: Code/main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- Code/main.py	(revision 274bcd99232d285de18abb8dd14cff1b321dfe69)
+++ Code/main.py	(revision a51687fb94153e0a91c2474095cc6810e97b25de)
@@ -236,4 +236,4 @@
         resp = client.message(command_if)
         if_else = 2
     if if_else == 0:
-        resp = client.message(resp['entities']['command:command'][0]['body'])
+        resp = client.message(resp['entities']['command:command'][0]['body'])
\ No newline at end of file
